lib/api/src/grpc/proto/qdrant_internal_service.proto

```diff
@@ -4,12 +4,24 @@ package qdrant;
 option csharp_namespace = "Qdrant.Client.Grpc";
 
 service QdrantInternal {
+  /*
+  Get current commit and term on the target node.
+   */
+  rpc GetCommitIndex (GetCommitIndexRequest) returns (GetCommitIndexResponse) {}
```

Comment:
Shall we rename this to `GetConsensusCommit` to be consistent with the preexisting things in this file?

Reply:
Sure.

Reply:
Done.